spec/server_spec.lua

```diff
@@ -50,16 +64,44 @@ describe("http.server module", function()
 		assert.truthy(cq:empty())
 		assert.spy(on_stream).was.called()
 	end
-	it("works with plain http 1.1", function()
+	it("works with plain http 1.1 using IP", function()
 		simple_test(false, 1.1)
 	end)
-	it("works with https 1.1", function()
+	it("works with https 1.1 using IP", function()
 		simple_test(true, 1.1)
 	end)
-	it("works with plain http 2.0", function()
+	it("works with plain http 2.0 using IP", function()
 		simple_test(false, 2.0)
 	end);
-	(require "http.tls".has_alpn and it or pending)("works with https 2.0", function()
+	(require "http.tls".has_alpn and it or pending)("works with https 2.0 using IP", function()
 		simple_test(true, 2.0)
 	end)
+	it("works with plain http 1.1 using UNIX socket domain", function()
+		local path = os.tmpname() .. ".socket"
```

Comment:
`os.tmpname` returns a different string each time.... you're generating a random file name to delete here.